ddtrace/profiling/exporter/http.py

```diff
@@ -146,17 +143,9 @@ def _get_tags(service):
         if env:
             tags["env"] = env
 
-        user_tags = os.getenv("DD_PROFILING_TAGS")
+        user_tags = os.environ.get("DD_TAGS", os.environ.get("DD_PROFILING_TAGS"))
```

Comment:
I'm not sure we should ignore `DD_PROFILING_TAGS` if `DD_TAGS` is there. I'd see them at being supplementary, where `DD_PROFILING_TAGS` would override the tags set in `DD_TAGS` if there's an overlap.

Reply:
:+1: what do you think about conflicting keys? Which should get precedence?

Reply:
also, do we envision using `DD_PROFILING_TAGS` long-term? Or is this just for the short-term?

Reply:
`DD_PROFILING_TAGS` should be used to add and possibly override tags from `DD_TAGS`.

I don't know if we're going to keep long term. I guess we can, it's a cool feature to have — worst case it's just useless.